cmd/bootstrap/group_replication.go

```diff
@@ -154,6 +156,15 @@ func (m *mysqlsh) getGTIDExecuted(ctx context.Context) (string, error) {
 	return result.Rows[0]["@@GTID_EXECUTED"], nil
 }
 
+func (m *mysqlsh) getGTIDPurged(ctx context.Context) (string, error) {
+	result, err := m.runSQL(ctx, "SELECT @@GTID_PURGED")
+	if err != nil {
+		return "", err
+	}
+
+	return result.Rows[0]["@@GTID_PURGED"], nil
```

Comment:
Not sure if it is worthwhile cause we already do the same thing in this file, but it would be better if we result validating instead of assuming that the key always exists.

```
	v, ok := result.Rows[0][<your-key>]
	if !ok {
		return "", fmt.Errorf(<some error message>)
	}
```

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/1003/commits/59dae4e401dcc4418aa87eed9ff0746471115eee